content/02.introduction.md

```diff
@@ -27,7 +27,7 @@ We also compare assorted approaches for applying knowledge graphs to solve biome
 Lastly, we conclude on the practicality of knowledge graphs and point out future applications that have yet to be explored.
 
 ![
-A metagraph (schema) of the heterogeneous network used in the Rephetio project [@doi:10.7554/eLife.26726].
-This undirected network depicts pharmacological and biomedical information.
+A subset of the knowledge graph used in the Rephetio project [@doi:10.7554/eLife.26726].
```

Comment:
If it's a subset of het.io then it probably needs to use the same directionality (if any) that het.io uses. I think some edges are directional but not all.